src/main/kotlin/org/jitsi/jibri/sipgateway/pjsua/PjsuaClient.kt

```diff
@@ -98,17 +98,24 @@ class PjsuaClient(
             command.add("--password=${pjsuaClientParams.sipClientParams.password}")
         }
 
-        if (pjsuaClientParams.sipClientParams.contact != null) {
+        if (!pjsuaClientParams.sipClientParams.contact.isNullOrEmpty()) {
```

Comment:
Is it possible pjsuaClientParams.sipClientParams.contact is null?

Reply:
Yes, if it is not set in the JSON request, its default value is `null` which comes from [SipClient.kt](https://github.com/jitsi/jibri/blob/master/src/main/kotlin/org/jitsi/jibri/sipgateway/SipClient.kt#L23-L53)

Reply:
Then that will produce a NPE?

Reply:
I didn't understand what you meant. They 3 possibilities:

- The `contact` is not in JSON which comes from API request. In this case, it is `null`
- The `contact` is in JSON but its value is an empty string, then it is `""` in the application to
- The `contact` is in JSON and its value is a string (not empty)

Reply:
If pjsuaClientParams.sipClientParams.contact is null, then doing `pjsuaClientParams.sipClientParams.contact.isNullOrEmpty()` in kotlin is ok and not producing a NPE? 
In java that will throw null pointer exception, that's why I'm asking.

Reply:
I also tested for all possible cases in my test environment, with `null`, empty string and non-empty string

Reply:
👍 sorry my kotlin knowledge

Reply:
It is new for me too :)
But I checked documents

Reply:
I also checked but when you told me it is fine :)

Reply:
I should have checked earlier ...